seqr/views/apis/feature_updates_api.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass, asdict
+
+from feedparser import parse
+from markdownify import markdownify as md
+
+import requests
+
+from seqr.views.utils.json_utils import create_json_response
+
+
+@dataclass
+class FeedEntry:
+    """Class representative of one Atom feed entry."""
+
+    author: str
+    author_link: str
+    link: str
+    markdown: str
+    published_datestr: str
+    title: str
+
+
+def get_feature_updates(request):
+    """
+    Fetches the feature-updates GitHub Discussion Atom feed, converts feed entries into markdown, and returns
+    markdown and information for each feed entry.
+    """
+    url = "https://github.com/broadinstitute/seqr/discussions/categories/feature-updates.atom"
```

Comment:
`url` and `timeout` should maybe be constants at the top of the file?

Reply:
added both as constants